src/styles/utils/_variables.scss

```
@@ -0,0 +1,9 @@
+$mainWidth: 100vw;
+$mainHeight: 100vh;
+
+$smallSize: 300px;
+$middleSize: 600px;
+$largeSize: 900px;
```

```suggestion
$mainWidth: 100vw;
$mainHeight: 100vh;

$smallSize: 300px;
$middleSize: 600px;
$largeSize: 900px;
```
Не понятно, почему ты задаешь контейнеру размеры окна, а вложенному блоку фиксированные размеры. У меня при запуске, часы вылазят за пределы окна браузера. Проинспектируй, как себя будут вести родительский и вложенный блоки при разных размерах окна.